root/etc/systemd/system/discovery-autostart.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=Start downloaded extension scripts
+Requires=nm-connected.service
```

Comment:
I see, ok. Well if `nm-online` fails to start, we are in trouble anyway, but good point.